### PR TITLE
fix: add typos in multipel files

### PR DESCRIPTION
**Pull request by rniii**

### CODE REVIEW

**Review by Vendicated — APPROVED** [review]
LGTM

**Review by Cynosphere — APPROVED** [review]
LGTM

**Comment by FieryFlames** [comment]
LGTM

**Review by AutumnVN — APPROVED** [review]
lgtm

File: scripts/generateReport.ts [hunk 1]
```diff
@@ -289,7 +289,7 @@ function runTime(token: string) {
             setTimeout(() => console.log("PUPPETEER_TEST_DONE_SIGNAL"), 1000);
         }, 1000));
     } catch (e) {
-        console.error("[PUP_DEBUG]", "A fatal error occurred");
+        console.error("[PUP_DEBUG]", "A fatal error occured");
```
Comment (nyakowint) [inline scripts/generateReport.ts:292]:
```suggestion
        console.error("[PUPPY_DEBUG]", "A fatal error occured woof woof");
```

**Review by FieryFlames — APPROVED** [review]
LGTM. In fact we need more typos

**Comment by ImLvna** [comment]
lgtm !

**Comment by Nuckyz** [comment]
I fixed the typo in the PR name.

**Comment by FieryFlames** [comment]
> I fxied the typo in the PR name.
I fixed the tpyo in your comment.

**Comment by wingio** [comment]
idk how i ended up here

**Comment by FieryFlames** [comment]
> idk how i ended up here

did you take a wrong turn trying to get to Aliucord

**Comment by wingio** [comment]
I've been trying to find the pasta aisle

**Comment by nyakowint** [comment]
https://github.com/Vendicated/Vencord/assets/24845294/1f6c27cd-e515-48a7-ad65-2c78ae128c5e

**Comment by nyakowint** [comment]
> I've been trying to find the pasta aisle

Don't you mean the soup store?

**Comment by FieryFlames** [comment]
> I've been trying to find the pasta aisle

oh ok pasta is in aisle #65

**Comment by nyakowint** [comment]
Guys can we get like 72 more prs so itll be literally 1984 again

**Comment by MrDiamondDog** [comment]
LGTM

**Comment by rniii** [comment]
looks gay to me

**Comment by nyakowint** [comment]
looks guh to me 

https://github.com/Vendicated/Vencord/assets/24845294/3591b04e-7d66-4298-8cf5-6762b9da1ca2

**Comment by Vendicated** [comment]
this pr is a certified homosexual 
grafik

**Review by wingio — APPROVED** [review]

**Comment by nyakowint** [comment]
> this pr is a certified homosexual grafik

How did sleepy joe get in here?????!?!? hes supposed to be chasing 淇淋 trucks

**Review by TheKodeToad — APPROVED** [review]

**Review by sunnniee — APPROVED** [review]

**Comment by Vendicated** [comment]
okay good to merge now

**Pull request closed without merging**